data/ecosystem/vendors.yaml

```diff
@@ -6,7 +6,7 @@
   commercial: true
 - name: AppDynamics (Cisco)
   nativeOTLP: true
-  url: https://docs.appdynamics.com/latest/en/application-monitoring/cisco-appdynamics-for-opentelemetry
+  url: https://docs.appdynamics.com/latest/en/application-monitoring/splunk-appdynamics-for-opentelemetry
```

Comment:
@tigrannajaryan @mtwo @pellared - can one of your approve of this URL change?

Also, should the name qualification be changed to Splunk as well?